civihr_default_theme/templates/contrib/views-slideshow-controls-text-next.tpl.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @file
+ * Views Slideshow: Template for text control - next.
+ *
+ * - $variables: Contains theme variables.
+ * - $classes: Text control classes.
+ * - $vss_id: The Views Slideshow unique id.
+ *
+ * @ingroup vss_templates
+ */
+?>
+<span id="views_slideshow_controls_text_next_<?php print $vss_id; ?>" class="<?php print $classes; ?>">
```

Comment:
This is more of a question:

How is this file loaded into the slide show? Where do the `$vss_id` and `$classes` values come from?

Reply:
This is a override of the original tpl file. Which has this content
```
<?php

/**
 * @file
 * Views Slideshow: Template for text control - next.
 *
 * - $variables: Contains theme variables.
 * - $classes: Text control classes.
 * - $vss_id: The Views Slideshow unique id.
 *
 * @ingroup vss_templates
 */
?>
<span id="views_slideshow_controls_text_next_<?php print $vss_id; ?>" class="<?php print $classes; ?>">
  <a href="#" rel="next"><?php print t('Next'); ?></a>
</span>
```

I have just replaced the `anchor` tags content with `Arrows`.